<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OneCurve</class>
 <widget class="QWidget" name="OneCurve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 8pt &quot;Agency FB&quot;;</string>
  </property>
  <widget class="QWidget" name="menubox" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="OneListWidget" name="list_curves">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="curveZero">
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 170, 0, 50);</string>
        </property>
        <property name="text">
         <string>Zero</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="curves_show_data">
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>DATA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refernceline">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>RL</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="axis_startpoint">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>20</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>start:(0,0)</string>
   </property>
  </widget>
  <widget class="QWidget" name="axis_x" native="true">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>210</y>
     <width>150</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>70</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="curve_max_x">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 8pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>StopX:65535</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="axis_lockX">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 8pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>LC</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="limit_X">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>80</horstretch>
            <verstretch>20</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 8pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>Max_x:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_maxX">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 8pt &quot;Agency FB&quot;;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="axis_y" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>150</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>70</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="curve_max_y">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 8pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>StopX:65535</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="axis_lockY">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 8pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>LC</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="limit_Y">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>80</horstretch>
            <verstretch>20</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 8pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>Max_x:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_maxY">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 8pt &quot;Agency FB&quot;;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_chioedEditer" native="true">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>130</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>130</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>130</width>
     <height>90</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgba(255, 255, 255, 50);
font: 9pt &quot;Agency FB&quot;;</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>110</width>
      <height>82</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="choiced_index">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>15</height>
        </size>
       </property>
       <property name="text">
        <string>Index:#####</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="choiced_mvdata">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>mVData:#####</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_mvdata">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>18</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>18</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="choiced_X">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>X:#####</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_X">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>18</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>18</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="choiced_Y">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>Y:#####</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_Y">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>18</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>18</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OneListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">onelistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
